static/src/javascripts/projects/common/modules/commercial/article-body-adverts.js

```diff
@@ -108,7 +110,16 @@ define([
 
     function insertAdAtPara(para, name, type) {
         var ad = createSlot(name, type);
-        para.parentNode.insertBefore(ad, para);
+        if (detect.isBreakpoint({max: 'tablet'})) {
+            // Insert ad using steady page
+            // To avoid jumping the user
+            steadyPage.insert(ad, function() {
```

Comment:
Hmm. We could abstract in spacefiller. The "DOM writer" would be fastdom by default but we could override it with one of our own at the calling site. Would need to provide a `write` function thought. WDYT?